blog/arquillian-cube-1.0.0.textile

```diff
@@ -0,0 +1,19 @@
+---
+date: 2017-03-24
+author: lordofthejars
+---
+
+p. New version of Arquillian Cube 1.0.0 has been released with one new feature and few bug fixes.
```

Comment:
Would be good to also explain that we switch versioning. Personally I would also add some excitement paragraph that we are finally out of never ending alpha :)